ngrx/ducks-schematics/src/collection/duck/files/__name@dasherize@if-flat__/__name@dasherize__.duck.ts.template

```diff
@@ -0,0 +1,18 @@
+import { Action, Ducksify, reducerFrom } from '@co-it/ngrx-ducks';
+import { <%= classify(name) %>State } from './<%= classify(name) %>.state';
+
+@Ducksify<<%= classify(name) %>State>({
+  initialState:
+})
+export class <%= classify(name) %> {
+  static reducer = reducer;
```

Comment:
I would remove the static member.
For now, I don't see the benefit, since the reducer is registered once in the `ActionReducerMap`.

If you want to add context to the reducer you also can rename the method to `counterReducer`.

Reply:
I use it mainly for code completion, it is easier to get the import automatically, but I removed it and renamed to `<name>Reducer`